invite/test/invite_bot.test.ts

```diff
@@ -14,33 +14,54 @@
  * limitations under the License.
  */
 
+import Knex from 'knex';
 import {mocked} from 'ts-jest/utils';
 
+import {setupDb} from '../src/setup_db';
+import {dbConnect} from '../src/db';
 import {Invite, InviteAction} from '../src/types';
 import {InvitationRecord} from '../src/invitation_record';
 import {InviteBot} from '../src/invite_bot';
 import {GitHub} from '../src/github';
 
+jest.mock('../src/db', () => {
```

Comment:
Again, not really needed for unit tests. Makes sense in e2e tests though

Reply:
I think it's necessary because `InviteBot` calls `dbConnect` in its constructor